### DISCUSSION TITLE
No rotation of the plot when xAxisDirection ist set from east to west

### DISCUSSION THREAD

**Discussion by BonitaBeichert** [mtex-toolbox/mtex#2066] [Ask Anything] [answered]
Hello, 

I have a problem with MTEX to analyse EBSD data. If I change the xAxisDirection to from east to west and leave the zAxisDirection outOfPlane the plots of the color maps dont rotate. A colleague has tested it on an older version of MTEX and there the plot turns.
Does anyone know the reason?

Thanks a lot!

`%% Specify File Names
% path to files
pname = 'Y:\Studierende\Bonita_Xuan_Beichert\EBSD\SD1-10';

% which files to be imported
fname = [pname '\SCA1_110_1_SD1-10.ctf'];

cs = {'notIndexed', crystalSymmetry('Fm-3m', [3.615 3.615 3.615], 'X||a', 'Z||c', 'mineral', 'Copper', 'color', 'light blue')};

% plotting convention
xAxisDir = 'east' 
zAxisDir = 'outOfPlane'

setMTEXpref('xAxisDirection', xAxisDir);
setMTEXpref('zAxisDirection', zAxisDir);

useMicronbar = 'off'
useCoordinates = 'on'
`

**Comment by kilir** [comment]
Hi,
what precisely do you mean with " color maps"?
Cheers,
Rüdiger

**Reply by BonitaBeichert** [reply]
Hi,

I have plotted fMapX with code code below. I thought the plot should turn 180 degrees when I just change the direction of the x axis but it stayes the same.

Thanks for the reply
Bonita

oriMapping = ipfHSVKey(cs{2});
%crystShape = crystalShape.hex(cs{2});

% IPFX
oriMapping.inversePoleFigureDirection = xvector;
colorIPFX = oriMapping.orientation2color(ebsd('Copper').orientations);

%% plot maps
fMapX = figure;
plot(ebsd('Copper'), colorIPFX, 'coordinates', useCoordinates, 'micronbar', useMicronbar, 'figSize', 'large')

![EBSD_Map](https://github.com/mtex-toolbox/mtex/assets/156414892/d47e7a64-fe20-4034-99c1-3aa4f1354ff8)

**Reply by kilir** [reply]
Can I assume you use mtex 5.10.x and not already 6.x?

**Reply by BonitaBeichert** [reply]
No unfortunately I already us MTEX 6.0

**Reply by kilir** [reply, marked as answer]
In that case everythign makes sense. The plotting conventions you set do not exist anymore and have no effect.

Please have a look here for the relevant information on how to set the plotting conventions.
https://mtex-toolbox.github.io/changelog#1

Also be aware that with 6.x the definition of upper hemisphere changed from "whereever positive z is pointing to" (<mtex 6) to "pointing out of the front side of your screen" (mtex >=6).

Cheers,

Rüdiger

**Reply by BonitaBeichert** [reply]
Okay thats very heplful. 

Thank you very much 
Bonita
